2024-04-14 15:58:15 INFO [32m pytest tests/backend/test_create_project.py::TestProjectCreateWithInvalidData::test_create_project_with_invalid_ids[Roles.PROJECT_VIEWER-MwGG 098] (setup)[0m
curl -X GET 'http://localhost:8111/authenticationTest.html?csrf' \
-H 'User-Agent: python-requests/2.31.0'\
-H 'Accept-Encoding: gzip, deflate'\
-H 'Accept: */*'\
-H 'Connection: keep-alive'\
-H 'Authorization: Basic OjEyOTQ0MTg2MjIwMjc0MjA1MjI=' \
-d ''
2024-04-14 15:58:15 INFO [32m pytest tests/backend/test_create_project.py::TestProjectCreateWithInvalidData::test_create_project_with_invalid_ids[Roles.PROJECT_VIEWER-MwGG 098][0m
curl -X POST 'http://localhost:8111/app/rest/projects' \
-H 'User-Agent: python-requests/2.31.0'\
-H 'Accept-Encoding: gzip, deflate'\
-H 'Accept: application/json'\
-H 'Connection: keep-alive'\
-H 'Content-Type: application/json'\
-H 'X-TC-CSRF-Token: 0e70fc7a-4785-432e-868c-255682af5537'\
-H 'Cookie: RecentLogin=; TCSESSIONID=3D404CAB88F62FED30D8ACE5CBCDC433'\
-H 'Content-Length: 110'\
-H 'Authorization: Basic OjEyOTQ0MTg2MjIwMjc0MjA1MjI=' \
-d '{"parentProject": {"locator": "_Root"}, "name": "friend", "id": "MwGG 098", "copeAllAssociatedSettings": true}' 

2024-04-14 15:58:15 INFO 	RESPONSE:
STATUS_CODE: [31m500[0m
DATA: [31mError has occurred during request processing, status code: 500 (Internal Server Error).
Details: jetbrains.buildServer.serverSide.InvalidIdentifierException: Project ID "MwGG 098" is invalid: contains unsupported character ' '. ID should start with a latin letter and contain only latin letters, digits and underscores (at most 225 characters).
Error occurred while processing this request.[0m